net-analyzer/tcpreplay/tcpreplay-999999.ebuild

```diff
@@ -49,8 +48,8 @@ src_configure() {
 	# By default it uses static linking. Avoid that, bug 252940
 	econf \
 		$(use_enable debug) \
-		$(use_with pcapnav pcapnav-config /usr/bin/pcapnav-config) \
-		$(use_with tcpdump tcpdump /usr/sbin/tcpdump) \
+		$(use_with pcapnav pcapnav-config "${EPREFIX}"/usr/bin/pcapnav-config) \
```

Comment:
`ESYSROOT`?

Reply:
This is tricky - it wants to run the binary, so let's call it "${BROOT}" for -config, and ${ESYSROOT} for tcpdump. Someone can file a bug if we need to do some meddling to pass the right parameters in a cross case.